internal/docstore/driver/util.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package driver
+
+import "github.com/google/uuid"
+
+// UniqueString generates a string that is unique with high probability.
+// Driver implementations can use it to generate keys for Create actions.
```

Comment:
Nit: not sure if this is worth factoring out.

Reply:
It is. Pretty much every driver will want this capability. Even though it's one line of code, it answers the following questions that a new driver author might have (I go through this in my head whenever I have this problem):
- How should I generate something that's really unique?
- Should it be a string? An int?
- Will crypto randomness be too slow?
- Maybe I should include time, so they're roughly sortable by time?
- I've heard of UUIDs. Is there a Go package that creates them for me?
- https://godoc.org/?q=uuid says there are a lot. What's a good one?